core/include/detray/propagator/actors/parameter_transporter.hpp

```diff
@@ -64,8 +64,9 @@ struct parameter_transporter : actor {
             const free_matrix_t correction_term =
                 matrix::identity<free_matrix_t>() + path_correction;
 
-            return free_to_bound_jacobian * correction_term *
-                   stepping.transport_jacobian() * bound_to_free_jacobian;
+            return free_to_bound_jacobian *
+                   (correction_term *
+                    (stepping.transport_jacobian() * bound_to_free_jacobian));
```

Comment:
Did you analyse the impact on physics performance of this change?

Reply:
I can open a test branch on traccc. Is that what you mean? In the end, I think, the floating point issues came from the usage of fma, as you suggested, and only changed things in the detray tests for the `cylinder2D` shape (as far as I can tell). This shape is not currently used in traccc

Reply:
Okay, then I am happy to accept this!

Reply:
Single precision tests in just release mode seem to run fine, also locally